cypress/support/commands.ts

```diff
@@ -228,3 +228,20 @@ export function addToCollection(path: string, organizationName: string, collecti
   cy.get('mat-option').contains(collectionDisplayName).click();
   cy.get('[data-cy=addEntryToCollectionButton]').should('not.be.disabled').click();
 }
+export function snapshot() {
+  cy.get('[data-cy=dockstore-snapshot-locked]').should('have.length', 0);
+  // The buttons should be present
+  cy.get('[data-cy=dockstore-request-doi-button]').its('length').should('be.gt', 0);
+  cy.get('[data-cy=dockstore-snapshot]').its('length').should('be.gt', 0);
+
+  cy.get('[data-cy=dockstore-snapshot-unlocked]').its('length').should('be.gt', 0);
+
+  cy.get('[data-cy=dockstore-snapshot]').first().click();
+
+  cy.get('[data-cy=snapshot-button]').click();
+
+  cy.wait(250);
```

Comment:
Was this necessary when you tried it? In theory, we generally shouldn't have to do it any more.

Reply:
Tested and confirmed we didn't need the wait anymore. Removed.